main.bundle.js

```diff
@@ -4572,16 +4572,16 @@
 /* 2 */
 /***/ (function(module, exports) {
 
-	module.exports = function(module) {
-		if(!module.webpackPolyfill) {
-			module.deprecate = function() {};
-			module.paths = [];
-			// module.parent = undefined by default
-			module.children = [];
-			module.webpackPolyfill = 1;
-		}
-		return module;
-	}
+	module.exports = function(module) {

```

Comment:
I think so, but if we ignore it then we have to make adjustments to our package.json, like the postinstall script I was trying to add a couple days ago. I couldn't get that to work, which is why it isn't ignored yet.